{% if cookiecutter.library_prefix %}{{ cookiecutter.library_prefix | capitalize }}_CircuitPython{% else %}CircuitPython_Org{% endif %}_{{ cookiecutter.library_name|replace(" ", "_")}}/README.rst

```diff
@@ -1,18 +1,18 @@
-{% if cookiecutter.library_prefix -%}
-    {% set prefix = cookiecutter.library_prefix | capitalize + "_" -%}
-    {% set repo_name = prefix + "CircuitPython_" + cookiecutter.library_name -%}
-{% else -%}
-    {% set prefix = '' -%}
-    {% set repo_name = "CircuitPython_Org_" + cookiecutter.library_name -%}
-{% endif -%}
+{%- if cookiecutter.library_prefix -%}
+    {%- set repo_name = (cookiecutter.library_prefix | capitalize) -%}
+    {%- set repo_name = repo_name + '_CircuitPython_' -%}
+    {%- set repo_name = repo_name + cookiecutter.library_name | replace(" ", "_") -%}
+{%- else -%}
+    {%- set repo_name = 'CircuitPython_Org_' + cookiecutter.library_name | replace(" ", "_") -%}
+{%- endif -%}
 
 {% set full_repo_name = cookiecutter.github_user + "/" + repo_name -%}
 {%- set pypi_name = cookiecutter.library_name|lower|replace("_", "-") -%}
 Introduction
 ============
 
 {% if cookiecutter.sphinx_docs | lower in ["yes", "y"] %}
-.. image:: https://readthedocs.org/projects/{% if cookiecutter.library_prefix %}{{ cookiecutter.library_prefix | lower | replace("_", "-")}}-{% endif %}circuitpython-{{ cookiecutter.library_name | lower }}/badge/?version=latest
+.. image:: https://readthedocs.org/projects/{% if cookiecutter.library_prefix %}{{ cookiecutter.library_prefix | lower | replace("_", "-")}}-{% endif %}circuitpython-{{ cookiecutter.library_name | lower | replace(" ", "-") }}/badge/?version=latest
 {%- if cookiecutter.target_bundle == 'Adafruit' %}
     :target: https://circuitpython.readthedocs.io/projects/{{ cookiecutter.library_name | lower }}/en/latest/
```

Comment:
The target has spaces in it when I ran the cookiecutter.

```
.. image:: https://readthedocs.org/projects/adafruit-circuitpython-my-amazing-library/badge/?version=latest
    :target: https://circuitpython.readthedocs.io/projects/my amazing library/en/latest/
    :alt: Documentation Status
```

Reply:
Thanks for taking a look, I will make these changes tonight.

Reply:
I think I've gotten all of those resolved with the latest commits.